Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -5,11 +5,11 @@ import PaymentsCore
 #endif
 
 /// PayPal Paysheet to handle PayPal transaction
+/// encapsulates instance to communicate with nxo
 public class PayPalClient {
```

Comment:
This is a public facing docstring, so it's message should be helpful to a merchant. We want to obfuscate "nxo" details from a merchant, and have them not even need to know what NXO is at all.

Reply:
I notice a lot of our other public classes are missing docstrings, so I created a [JIRA](https://engineering.paypalcorp.com/jira/browse/DTNOR-574) for us to tackle this holistically. 🏈